flang/test/Driver/save-mlir-temps.f90

```diff
@@ -51,9 +51,9 @@
 ! Content to check from the MLIR outputs
 !--------------------------
 ! MLIR-FIR-NOT: llvm.func
-! MLIR-FIR: func.func @{{.*}}main() {
+! MLIR-FIR: func.func @{{.*}}main(){{.*}}{
```

Comment:
What do these extra braces capture?

Reply:
This was added due to failing the test when it was a default parameter. I do not see how  having `{{.*}}` instead of a whitespace should be an issue if no parameters are checked.

Reply:
Could you explain why it was failing? I just don't see how this particular change is related to what this PR is implementing. It would help if you included code "before" and "after" this PR.

Reply:
Since this is target dependent, frame pointer will be added or not. It is either adding `{{.*}}` or adding `-fomit-frame-pointer`.